WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -195,7 +189,7 @@ public async void BanDoubleSpenders(object? sender, Transaction tx)
 	}
 
 	private bool IsWasabiCoinJoin(uint256 txId, Transaction tx) =>
-		CoinJoinIdStore.Contains(txId) || IsFinishedCoinJoin(txId) || IsWasabiCoinJoinLookingTx(tx);
+		IsFinishedCoinJoin(txId) || IsWasabiCoinJoinLookingTx(tx);
```

Comment:
I'd say this change is the riskier here. We added this because we had problems with double spending. `IsWasabiCoinJoinLookingTx(tx)`, and problems we had might be cause with the overloaded coordinator so it might be safe yet I'm not 100% sure